src/fs.rs

```diff
@@ -546,9 +546,16 @@ fn path_fd() {
 ///
 /// # Note
 ///
-/// Landlock rules operate on file descriptors, not paths. This is a helper to create rules based on paths. **It
-/// silently ignores paths that cannot be opened**, and automatically adjusts access rights according to file types when
-/// possible.
+/// This is a helper to create rules based on paths. However, Landlock rules operate on file descriptors, not paths.
+/// When defining a rule based on a path, a file should be present at the provided path. Otherwise, Landlock
+/// **silently ignores paths that cannot be opened by default**, as it deems such rules to be incompatible.
```

Comment:
> Landlock **silently ignores paths that cannot be opened by default**

This is false. Landlock (the kernel feature) cannot ignore paths because it only takes file descriptors. Only the `path_beneath_rules()` helper ignores non-existing paths.